app/controllers/courses/age_range_controller.rb

```diff
@@ -43,6 +27,44 @@ def update
 
   private
 
+    def update_age_range_param
+      if age_from_param.present? && age_to_param.present? && age_range_is_other?
+        params[:course][:age_range_in_years] = "#{age_from_param}_to_#{age_to_param}"
+      elsif age_range_is_other?
+        params[:course][:age_range_in_years] = nil
+      end
+    end
+
+    def build_errors
+      if age_range_is_other? && (age_from_param.blank? || age_to_param.blank?)
+        {
+          age_range_in_years: ["Enter an age for both from and to"],
+          age_range_in_years_from: ["Enter an age"],
+          age_range_in_years_to: ["Enter an age"],
+        }
+      end
```

Comment:
While we're here, is this right?? Doesn't this mean that if either is blank, both fields will be flagged with this message?

Reply:
Yeah looks like it @misaka - will fix

Reply:
@misaka - spoke to Dan and this will be fixed as part of another ticket.